docs/adguard-for-mac/overview.md

```diff
@@ -3,4 +3,153 @@ title: Features overview
+
+**3. AdGuard Popup Blocker**
+Prevents all these annoying popup windows from opening when you view webpages. Blocks popup ads on web pages.
+
+One of the important features of AdGuard for Mac is that the application can work as a cross-browser userscript manager. You can easily add any scripts you like and manage existing ones without having to switch browsers every time.
+
+## Network
+
+![Network](https://cdn.adguard.com/content/kb/ad_blocker/mac/NetworkFiltering.jpg)
+    
+This last panel is dedicated to Network filtering, and you will find extra functionalities here. We do recommend to enable the first two ones: "Automatically filter applications" and "Filter HTTPS protocol". Those are an important added precaution to better filter your web space. HTTP is the primary protocol for transmitting information over the Internet. Most data including ads is transferred via it, but it has one disadvantage — the connection is unencrypted. Thanks to the HTTPS filtering feature AdGuard has, not only HTTP traffic is filtered, but also HTTPS traffic (S stands for ‘secure’).
+ 
+There are also the options of "Do not filter websites with EV certificates" and "Use AdGuard as an HTTP Proxy". Extended Validation (EV) SSL Certificates offer a stronger safety guarantee; owners of such websites have to pass a thorough and globally standardized identity verification process defined by EV guidelines. Which is the reason why some users trust websites with such certificates and prefer not to filter them.
+ 
+As for the second option — well, yes, in fact you can use AdGuard as a regular HTTP proxy server. All traffic coming through it will be filtered. This proxy will also be available on your local network, so you may use it for traffic filtering on other devices that are able to reach the proxy directly. To filter traffic on another device connected to proxy you will need to install an AdGuard root certificate there. After you configure proxy, open your browser and go to [this page](http://local.adguard.org/cert).
+
+## Others
+
+There are other useful AdGuard options that shouldn't go unnoticed in this article, since they add much to user experience. 
+
+### License
+
+First, there's the License management tab which you can open from the dropdown main menu (the one that opens once you click on the gear icon on the upper right of the main window). From this tab you can see which license type you are using, check its status and validity period. 
+    
+![License](https://cdn.adguard.com/content/kb/ad_blocker/mac/License.jpg)
+
+In this same tab you can refresh your license status, renew your license or reset the license key using the corresponding buttons.
+
+### Support
+
+Click on the gear icon in the upper right corner, select "Support", and you will be able to send a bug report or a feature request to our support team. Be sure to add your real actual email though, because without it the message won't be sent. 
```

Comment:
```suggestion
Click the gear icon in the upper right corner, select "Support", and you will be able to send a bug report or a feature request to our support team. Be sure to add your real actual email though, because without it the message won't be sent. 
```